app/views/podcasts/feed_new.xml

```diff
@@ -1,69 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Huh, seems like I unintentionally committed this when working on the importer UI.  Whoops!